Tests/LifecycleTests/ServiceLifecycleTests.swift

```diff
@@ -91,6 +91,55 @@ final class ServiceLifecycleTests: XCTestCase {
         XCTAssertEqual(item.state, .shutdown, "expected item to be shutdown")
     }
 
+    func testStartAndWaitShutdownWithSignal() {
+        if ProcessInfo.processInfo.environment["SKIP_SIGNAL_TEST"].flatMap(Bool.init) ?? false {
+            print("skipping testStartAndWaitShutdownWithSignal")
```

Comment:
You might be able to use [`XCTSkipUnless`](https://developer.apple.com/documentation/xctest/3521326-xctskipunless) here. 

Example: https://github.com/vapor/postgres-nio/blob/master/Tests/PostgresNIOTests/PerformanceTests.swift#L13